docs/configuration/index.md

```diff
@@ -213,7 +213,7 @@ Redocly CLI supports one `http` header per URL.
 
 Here is an example for adding header definitions:
 
-```yaml
+```text
```

Comment:
What's the difference? The example below is a proper `yaml` as far as I can judge.

Reply:
The syntax highlighter thinks that the `/*` bits are a comment and turns half of it grey! Our yaml highlight is so plain that the text isn't much different so I flipped it.